sql/clients_last_seen_raw_v1.sql

```diff
@@ -0,0 +1,74 @@
+-- Equivalent to, but more efficient than, calling udf_bitmask_range(1, 28)
+CREATE TEMP FUNCTION bitmask_lowest_28() AS (0x0FFFFFFF);
+--
+CREATE TEMP FUNCTION shift_one_day(x INT64) AS (IFNULL((x << 1) & bitmask_lowest_28(), 0));
+--
+CREATE TEMP FUNCTION combine_days(prev INT64, curr INT64) AS (IFNULL(shift_one_day(prev), 0) + IFNULL(curr, 0));
+--
+WITH
+  _current AS (
+  SELECT
+    -- In this raw table, we capture the history of activity over the past
+    -- 28 days for each usage criterion as a single 64-bit integer. The
+    -- rightmost bit represents whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_seen_bits,
+    -- For measuring Active MAU, where this is the days since this
+    -- client_id was an Active User as defined by
+    -- https://docs.telemetry.mozilla.org/cookbooks/active_dau.html
+    CAST(scalar_parent_browser_engagement_total_uri_count_sum >= 5 AS INT64) AS days_visited_5_uri_bits,
+    CAST(devtools_toolbox_opened_count_sum > 0 AS INT64) AS days_opened_dev_tools_bits,
+    DATE_DIFF(submission_date_s3, SAFE_CAST(SUBSTR(profile_creation_date, 0, 10) AS DATE), DAY) AS days_since_created_profile,
```

Comment:
I think it would be more clear to use `PARSE_DATE` here instead of `CAST`:
```suggestion
    DATE_DIFF(submission_date_s3, SAFE.PARSE_DATE("%F", SUBSTR(profile_creation_date, 0, 10)), DAY) AS days_since_created_profile,
```